### PR TITLE
KVID-671 - fix full screen issue replaying fabrik outstream

### PR DESCRIPTION
**Pull request by EdmundsBankai**
## Proposed changes
- List an overview of the changes included in this PR

## Dependencies
- Update this if the PR requires us to update the project’s dependencies

## Checklist
- [x] Tested locally
- [ ] Added new unit, integration or regression tests
- [ ] Updated docs

## Demo
- Add before and after screenshots and videos showcasing the changes

### CODE REVIEW

File: src/dynamics/ads_player/ads_player.js [hunk 1]
```diff
@@ -437,7 +438,8 @@ Scoped.define("module:Ads.Dynamics.Player", [
                                 this._video.src = URL.createObjectURL(blob);
                                 this._video.crossOrigin = "anonymous";
                                 this._video.muted = true;
-                                return this._video.play()
+                                this._video.setAttribute("playsinline", true);
```
Comment (jwellborn-kargo) [inline src/dynamics/ads_player/ads_player.js:441]: Why would this attribute change to begin with?

**Review by jwellborn-kargo — APPROVED** [review]
Just one question.

File: src/dynamics/ads_player/ads_player.js (hunk 1 above)
Comment (EdmundsBankai) [reply src/dynamics/ads_player/ads_player.js:441]: https://webkit.org/blog/6784/new-video-policies-for-ios/
https://stackoverflow.com/questions/75680106/how-to-play-a-video-in-the-background-on-ios-safari
safari ios has some security policy that causes the video to open in full screen

@jwellborn-kargo

Comment (EdmundsBankai) [reply src/dynamics/ads_player/ads_player.js:441]: oh, i think i misunderstood the question, its because this._video is not the same variable as video

**Review by maxblessed — APPROVED** [review]

**Pull request merged**